src/contracts/atlas/Escrow.sol

```diff
@@ -26,6 +27,7 @@ abstract contract Escrow is AtlETH {
     using EscrowBits for uint256;
     using CallBits for uint32;
     using SafetyBits for Context;
+    using SafeCall for address;
```

Comment:
I'm a bit confused by this, doesn't this override `call()` on any `address` in this contract? So it would also apply to all the `ExecutionEnvironment` calls as well?

Reply:
ohh okay, that makes sense. i was wondering why tests were passing here.

imo something to make this clearer would be good, i'm not a big fan of overloading builtin types at all really, so if we have to do it lets make it as unconfusing as possible. reminds me of ruby on rails when they monkey patched `minutes` and `seconds` onto integer 😭  `Time.now + 1.minutes`

Reply:
> Time.now + 1.minutes

lmao yikes

Reply:
Yeah this is much clearer https://github.com/FastLane-Labs/atlas/pull/272/commits/80f56767e7e6dbb992a9618523ae97070d89b5f7